### PR TITLE
Carbon Footprint

### PR DESCRIPTION
**Pull request by lalitkumardhanjani**

### CODE REVIEW

File: EmailCarbonEmission.cpp [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+using namespace std;
+
+class GmailClient {
+public:
+    GmailClient(const string& email, const string& password) : email(email), password(password) {
+        vmime::platform::setHandler<vmime::platforms::posix::posixHandler>();
+    }
+
+    void Connect_To_Gmail() {
+        try {
```
Comment (harshal0412) [inline EmailCarbonEmission.cpp]: store variable could be better named

File: EmailCarbonEmission.cpp [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            Fetch_Email_Counts();
+
+            store->disconnect();
+        } catch (const exception& e) {
+            cerr << "Error: " << e.what() << endl;
+        }
+    }
+
+    void Fetch_Email_Counts() {
+        cout << "Fetching and counting email types..." << endl;
```
Comment (harshal0412) [inline EmailCarbonEmission.cpp]: use more descriptive names

File: EmailCarbonEmission.cpp [hunk 3]
```diff
@@ -0,0 +1,65 @@
+#include <iostream>
+#include <vmime/vmime.hpp>
+#include <vmime/platforms/posix/posixHandler.hpp>
+
+using namespace std;
+
+class GmailClient {
```
Comment (JigyasaKarodiwal-ITT) [inline EmailCarbonEmission.cpp]: I am not understanding what is meant by gmail client here. Please give some more descriptive name to the class.

File: EmailCarbonEmission.cpp [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+            cout << "Successfully connected to Gmail server!" << endl;
+
+            Fetch_Email_Counts();
+
+            store->disconnect();
+        } catch (const exception& e) {
+            cerr << "Error: " << e.what() << endl;
+        }
+    }
+
```
Comment (JigyasaKarodiwal-ITT) [inline EmailCarbonEmission.cpp]: use comments to describe the working of the functions.

File: EmailCarbonEmission.cpp (hunk 3 above)
Comment (Kartikeybharti8) [inline EmailCarbonEmission.cpp]: Class name can should be more descriptive.

File: EmailCarbonEmission.cpp [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+using namespace std;
+
+class GmailClient {
+public:
+    GmailClient(const string& email, const string& password) : email(email), password(password) {
+        vmime::platform::setHandler<vmime::platforms::posix::posixHandler>();
+    }
+
+    void Connect_To_Gmail() {
```
Comment (Kartikeybharti8) [inline EmailCarbonEmission.cpp]: Use comments to represent what function is doing.

**Pull request closed without merging**